docs/regmap/adi_regmap_spi_engine.txt

```diff
@@ -425,11 +462,67 @@ OFFLOAD0_SDO_FIFO
 ENDREG
 
 FIELD
-[31:0] 0x????????
+[31:0] 0xXXXXXXXX
 OFFLOAD0_SDO_FIFO
 WO
 Offload SDO FIFO register. Writing to this register inserts an entry into the offload SDO FIFO.
 Writing to this register when the SDO FIFO is full has no effect and the written entry is
-discarded. Reading from this register always returns 0x00000000.
+discarded. Reading from this register always returns 0x0.
+ENDFIELD
+
+############################################################################################
+############################################################################################
+
+REG
+0x80
+CFG_INFO_0
+ENDREG
+
+FIELD
+[31:0] ''CFG_INFO_0''
+CFG_INFO_0
+RO
+Configuration Info.
+ENDFIELD
+############################################################################################
+############################################################################################
+
+REG
+0x81
+CFG_INFO_1
+ENDREG
+
+FIELD
+[31:0] 'CFG_INFO_1''
+CFG_INFO_1
+RO
+Configuration Info.
+ENDFIELD
+############################################################################################
+############################################################################################
+
+REG
+0x82
+CFG_INFO_2
+ENDREG
+
+FIELD
+[31:0] 'CFG_INFO_2''
+CFG_INFO_2
+RO
+Configuration Info.
 ENDFIELD
+############################################################################################
+############################################################################################
+
+REG
+0x83
+CFG_INFO_3
+ENDREG
 
+FIELD
+[31:0] ''CFG_INFO_3'"
```

Comment:
Double quote instead of single in the end
```
[31:0] ''CFG_INFO_3''
```
those cause the 
```
WARNING: Inline literal start-string without end-string.
```
error, since effectively you opened an inline literal and didn't close it.